composer.lock

```diff
@@ -210,24 +210,24 @@
         },
         {
             "name": "composer/composer",
-            "version": "1.10.6",
+            "version": "1.10.7",
```

Comment:
We don't want to update dependencies as a part of this change.
You can use `composer update -lock` to only update composer.lock hash without updating dependencies